src/main/java/com/google/sps/servlets/SearchMealServlet.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+
+@WebServlet("/search-meal")
+public class SearchMealServlet extends HttpServlet {
+
+    private ArrayList<String> dishes;
```

Comment:
You should first establish a common data model (i.e. how is a dish represented in your Java code) before adding logic like this.
If you use `String` to represent the entire object, you'll need to soon recycle your code to use Olga's `Meal` Java object representing the data instead.

Reply:
Of course I'll change it, I started my part earlier, thats why I used `String`.

Reply:
It's usually a good idea to clearly mark temporary solutions in the code. Some kinda comment, e.g.
```
TODO(sandatsian): replace String with Meal
```
would do.